packages/nextlove/src/nextjs-exception-middleware/with-exception-handling.ts

```diff
@@ -2,6 +2,7 @@ import { NextApiRequest, NextApiResponse } from "next"
 import { HttpException } from "./http-exceptions"
 
 export interface WithExceptionHandlingOptions {
+  errorLogger?: (error: Error, req: NextApiRequest) => void
```

Comment:
Curious why this is optional, should we mandate that all routes have error logging?

Reply:
Since we apply this handler globally, in our use case we only need to set this once in the global middleware.

See https://github.com/seamapi/seam-connect/pull/7794

Reply:
Nothing about this implementation appears specific to logging
```suggestion
  onError?: (error: Error, req: NextApiRequest) => void
```

Reply:
> Curious why this is optional, should we mandate that all routes have error logging?

Adding this as required would make this a breaking change, and I agree it's not clear this lib should require it. If we did want to make it required for a project, we can wrap this I think.